test/queried-for.spec.js

```diff
@@ -31,9 +31,7 @@ describe('queried-for', () => {
           'queried for',
           '#parent',
           'to satisfy',
-          <div className="old friendly">
-            <div id="child">Children</div>
-          </div>
+          <div id="child">Children</div>
```

Comment:
This is the bit that messes up the spike. The problem is that shallow renders on level and the full render keeps all the information. So this passes, but I think the previous behavior here makes more sense. It would be nice if we could make the two renders act similarly.